plugin/trino-example-plugin/src/main/java/com/example/ExampleColumnHandle.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.trino.spi.connector.ColumnHandle;
+import io.trino.spi.type.Type;
+
+import java.util.Objects;
+
+public class ExampleColumnHandle
+        implements ColumnHandle
+{
+    private final String name;
+    private final Type type;
+
+    @JsonCreator
+    public ExampleColumnHandle(
+            @JsonProperty("name") String name,
```

Comment:
We don't need `@JsonProperty` on constructor arguments anymore, since we configure Jackson to use parameter names. We'll remove them everywhere going forward.

Reply:
Cleaned up here and in the table handle.